templates/wizards/projects/asn1acn/createFilesList.cmake

```diff
@@ -38,11 +38,12 @@ function(tryCreateFiles fileNames)
 endfunction()
 
 function(appendPersistentListC)
-    set(persistentFiles ${ASN1SCC_C_DIR}/asn1crt.c ${ASN1SCC_C_DIR}/real.c)
```

Comment:
real.c is no longer generated? real/float related functions are included in asn1crt?

Reply:
yes, real.c is no longer generated
no, real/float related functions are included in asn1crt_encoding.h/.c